app/src/main/res/layout/dialog_radio_group.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we have dialog_radio_group and radio_button in commons already, you can use those. You might have to use the oldschool findviewbyid for referencing views to make it work well

Reply:
Fixed this.